packages/starlight-theme-ensnode/overrides/ThemeSelect.astro

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Curious about this empty file role? 🙃

Reply:
Basically this overrides the ThemeSelect to effectively show nothing where this would be - disabling selecting themes.